WORKFLOW-OSX-Machine-Setup-RoR.md

```diff
@@ -0,0 +1,87 @@
+# Basic OS X Machine Setup for RoR Development 
+####Pre-requisites
+1. Update to Yosemite(OSX 10.10)
+
+2. Install xCode 6.x & xCode Command Line Tools
```

Comment:
How about "install Homebrew"? People will need homebrew anyway, and installing Homebrew on a fresh OSX will walk you through Xcode CLI tools installation.

source: came from a fresh Yosemite setup recently :)